.ci/build.v1.yml

```diff
@@ -56,28 +57,76 @@ parameters:
+    displayName: '${{ parameters.displayName }} linux'
+    timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+    continueOnError: ${{ eq(parameters.continueOnError, 'true') }}
+    dependsOn: ${{ parameters.dependsOn }}
+    pool:
+      name: ${{ parameters.linuxAgentPoolName }}
+      vmImage: ${{ parameters.linuxImage }}
+      ${{ if ne(parameters.linuxImageOverride, '') }}:
+        demands:
+        - ImageOverride -equals ${{ parameters.linuxImageOverride }}
+    steps:
+    - template: build.steps.v1.yml
+      parameters:
+        name: ${{ parameters.name }}
+        displayName: ${{ parameters.displayName }}
+        timeoutInMinutes: ${{ parameters.timeoutInMinutes }}
+        dependsOn: ${{ parameters.dependsOn }}
+        initSteps: ${{ parameters.initSteps }}
+        preBuildSteps: ${{ parameters.preBuildSteps }}
+        postBuildSteps: ${{ parameters.postBuildSteps }}
+        masterBranchName: ${{ parameters.masterBranchName }}
+        installAppleCertificates: ${{ parameters.installAppleCertificates }}
+        submodules: ${{ parameters.submodules }}
+        areaPath: ${{ parameters.areaPath }}
+        runChecks: ${{ parameters.runChecks }}
+        continueOnError: ${{ parameters.continueOnError }}
+        publishJob: ${{ parameters.publishJob }}
+        publishOutputSuffix: ${{ parameters.publishOutputSuffix }}
+        signListPath: ${{ parameters.signListPath }}
+        linuxAgentPoolName: ${{ parameters.linuxAgentPoolName }}
+        macosAgentPoolName: ${{ parameters.macosAgentPoolName }}
+        windowsAgentPoolName: ${{ parameters.windowsAgentPoolName }}
+        linuxImage: ${{ parameters.linuxImage }}
+        macosImage: ${{ parameters.macosImage }}
+        windowsImage: ${{ parameters.windowsImage }}
+        windowsImageOverride: ${{ parameters.windowsImageOverride }}
+        mono: ${{ parameters.mono }}
+        xcode: ${{ parameters.xcode }}
+        dotnet: ${{ parameters.dotnet }}
+        dotnetStable: ${{ parameters.dotnetStable }}
+        cake: ${{ parameters.cake }}
+        apiTools: ${{ parameters.apiTools }}
+        xharness: ${{ parameters.xharness }}
+        tools: ${{ parameters.tools }}
+        cakeTemplatesBranch: ${{ parameters.cakeTemplatesBranch }}
+        buildType: ${{ parameters.buildType }}
+        steps: ${{ parameters.steps }}
+        verbosity: ${{ parameters.verbosity }}
+        configuration: ${{ parameters.configuration }}
+        validPackagePrefixes: ${{ parameters.validPackagePrefixes }}
+        artifactsPath: ${{ parameters.artifactsPath }}
+        cakeTarget: ${{ parameters.cakeTarget }}
+        cakeFile: ${{ parameters.cakeFile }}
+        cakeExtraArgs: ${{ parameters.cakeExtraArgs }}
+        forceBuild: ${{ parameters.forceBuild }}
+        namesFilter: ${{ parameters.namesFilter }}
+        targetsFilter: ${{ parameters.targetsFilter }}
+
 
-    displayName: ${{ parameters.displayName }}
+- ${{ if ne(parameters.macosImage, '') }}:
+  - job: macos
```

Comment:
Is this name needed? This is used as a [template for Xamarin.Essentials](https://github.com/xamarin/Essentials/blob/main/azure-pipelines.yml#L121) and because we're repeating it a couple of times this now breaks us because this is seen as not unique anymore